ledger/ledger-utxo/src/main/kotlin/net/corda/v5/ledger/utxo/ContractVerificationFailureReason.kt

```diff
@@ -0,0 +1,17 @@
+package net.corda.v5.ledger.utxo
+
+import net.corda.v5.base.annotations.CordaSerializable
+import net.corda.v5.base.annotations.DoNotImplement
+
+/**
+ * Defines a contract verification failure reason.
+ *
+ * @property command The contract [Command] that caused the verification failure.
+ * @property cause The underlying cause of the verification failure.
+ */
+@CordaSerializable
+@DoNotImplement
+interface ContractVerificationFailureReason {
```

Comment:
Small thing, does the "reason" add anything here or is `ContractVerificationFailure` enough?

Reply:
Yes, on balance, I think `ContractVerificationFailure` is enough.